File: src/test/java/edu/hm/hafner/edu/hm/hafner/analysis/edu/hm/hafner/analysis/assertions/IssuesSoftAssertions.java
package edu.hm.hafner.edu.hm.hafner.analysis.edu.hm.hafner.analysis.assertions;

import org.assertj.core.api.SoftAssertions;
import edu.hm.hafner.analysis.Issues;

public class IssuesSoftAssertions extends SoftAssertions {
    public IssuesAssertions assertThat(final Issues actual) {
        return proxy(IssuesAssertions.class, Issues.class, actual);
    }
}

File: src/test/java/edu/hm/hafner/analysis/assertj/kotlin/IssuesSoftAssertions.kt
package edu.hm.hafner.analysis.assertj

import edu.hm.hafner.analysis.Issues

open class IssuesSoftAssertions : SoftAssertions() {
    fun assertThat(actual: Issues) : IssuesAssert {
        return proxy(IssuesAssert::class.java, Issues::class.java, actual)
    }
}